planetarium/NineChronicles
# Change boss resources.

### Description

1. Apply new boss spine resource.
2. Fix mismatched resources.

### How to test

1. Play several boss stage and check if the new victory pose of the bosses.
(Changed bosses : 201005, 202007, 203007, 205007, 206007)
2. Go to '202005' folder and check if the resource is correctly matched.

### Required Reviewers

@Namyujeong or anyone.
